src/extras/render-passes/render-pass-compose.js

```diff
@@ -13,6 +11,11 @@ import { ShaderGenerator } from '../../scene/shader-lib/programs/shader-generato
 // before the tone-mapping.
 
 const fragmentShader = /* glsl */ `
+
+    #include "tonemappingPS"
```

Comment:
Oh man, I wish we didn't have that `PS` suffix. 😬

Reply:
agreed :)